chpl/chpl-service/src/test/java/old/gov/healthit/chpl/validation/listing/reviewer/ChplNumberReviewerTest.java

```diff
@@ -60,6 +61,9 @@
     @Autowired
     private MessageSource messageSource;
 
+    @Spy
```

Comment:
Are we trying to maintain the "old" tests.  I think this is a good reason for us to remove them from the project, but this might be a discussion for the Dev team.

Reply:
To me this test is close-ish to working as mock-only so maybe the thing that I should do is actually make it work like that and move it into the new tests.